tests/contrib/django_util/test_django_models.py

```diff
@@ -19,17 +19,15 @@
 
 import base64
 import pickle
-
-from tests.contrib.django_util.models import CredentialsModel
-
-import unittest2
+import unittest
 
 from oauth2client import _helpers
 from oauth2client.client import Credentials
 from oauth2client.contrib.django_util.models import CredentialsField
+from tests.contrib.django_util.models import CredentialsModel
```

Comment:
Move this line back for now, to appease `flake8`.  I'll take care of it in another PR today, since the `flake8` config and a few more files need to be updated.